publisher-sdk/src/main/java/com/criteo/publisher/CriteoBannerView.java

```diff
@@ -49,6 +49,13 @@
   @Nullable
   private CriteoBannerEventController criteoBannerEventController;
 
+  /**
+   * Will be used for server side bidding and in-house auction
```

Comment:
I find weird to have a comment at the future tense. What do you think about rephrasing it ? Or if this API is not yet ready for being used by publishers, maybe it is worth mentioning that it should not be used (maybe even mark it with @Internal ??)

Also, don't you mind adding a comment on the other constructor explaining that it is intended for standalone ?